jaraco/abode/cli.py

```diff
@@ -530,3 +530,7 @@ def main():
 
     with _log_errors_and_logout(_create_client_instance(args)) as client:
         Dispatcher(client, args).dispatch()
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Let's omit this change. The recommended way to invoke the CLI is `py -m jaraco.abode`.

Reply:
Also, the diffcov failure should go away when removing this change.

Reply:
Reverted the change